go.mod

```diff
@@ -245,4 +245,6 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // automatically, so we need to add it manually.
 replace github.com/golang-migrate/migrate/v4 => github.com/lightninglabs/migrate/v4 v4.18.2-9023d66a-fork-pr-2
 
-go 1.23.6
+replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.19.0-beta
+
+go 1.23.9
```

Comment:
Could bump golang in the `litrpc`, `autopilotserverrpc`, `perms` & `tools` packages as well.